service/riotapi.py

```diff
@@ -5,7 +5,10 @@ def get_puuid(game_tag, tag_line, api_key):
     get_puuid_url = str.format("https://americas.api.riotgames.com/riot/account/v1/accounts/by-riot-id/{0}/{1}?api_key={2}",
                                game_tag, tag_line, api_key)
     puuid_response_dict = get_response_json(get_puuid_url)
-    return puuid_response_dict.get("puuid")
+    if "status" in puuid_response_dict:
+        return puuid_response_dict
```

Comment:
I come from a Java world, so this scares me - having a single method that returns 2 totally separate things (dict vs string) seems like a maintenance nightmare.

Reply:
I am cool with discussing alternatives.